HW_1/task_5.py

```diff
@@ -0,0 +1,21 @@
+"""
+Напишите программу, которая принимает на вход координаты двух точек и находит расстояние между ними в 2D пространстве.
+Результат округлить до сотых.
+
+Ввод: четыре значения типа <int>
+Вывод: значение типа <float>
+
+Пример:
+
+4 10
+11 5
+9.22
+"""
+
+Ax = int(input('Введите координату х точки A: '))
+Ay = int(input('Введите координату y точки A: '))
+Bx = int(input('Введите координату x точки B: '))
+By = int(input('Введите координату y точки B: '))
+
+point_distance = ((Ax-Bx)**2 + (Ay-By)**2)**0.5
```

Comment:
Не забывайте приводить к PEP8.
```suggestion
point_distance = ((Ax - Bx) ** 2 + (Ay - By) ** 2) ** 0.5
```